Promscale docs (#654)
timescale/docs.timescale.com-content: tutorials/promscale-install.md

> ## 3. Up and Running: Install Prometheus, Promscale and TimescaleDB [](up-and-running)

Reviewer: Fix hierarchy of headlines

Reviewer 2: fixed

We recommend four methods to setup and install Promscale:
1. [TOBS - The Observability Stack for Kubernetes ][tobs-github] (recommended for kubernetes deployments)

> 2. Docker (instructions detailed below) 

Reviewer: This should be first on the list?

3. [Helm][promscale-helm-chart]
4. [Bare Metal][promscale-baremetal-docs]

> :TIP: See the [Promscale github installation guide][promscale-github-installation] for more information on installation options.

For demonstration purposes, we will use Docker to get up and running with Promscale.

The easiest way to get started is by using Docker images. Make sure you have Docker installed on your local machine ([Docker installation instructions][docker]).

The instructions below have 4 steps:
1. [Install TimescaleDB](#install-timescaledb)
2. [Install Promscale](#install-promscale)
3. [Install node_exporter](#install-node-exporter)
4. [Install Prometheus](#install-prometheus)

>:WARNING: The instructions below are local testing purposes only and should not be used to set up a production environment.

> ### Step 1: Install TimescaleDB [](install-timescaledb)

Reviewer: All of the previous pages used decimal sections "1.1", "2.3", etc.  Please use this here to stay consistent (Or change the top-level headline numbering system)

Reviewer 2: Good catch, fixed

First, let's create a network specific to Promscale and TimescaleDB:

```bash
docker network create --driver bridge promscale-timescaledb
```

Secondly, let's install and spin up an instance of TimescaleDB in a docker container.  This is where Promscale will store all metrics data scraped from Prometheus targets. 

We will use a docker image which has the`promscale` PostgreSQL extension already pre-installed:

```bash
docker run --name timescaledb \
    --network promscale-timescaledb \
    -e POSTGRES_PASSWORD=<password> -d -p 5432:5432 \
    timescaledev/timescaledb-ha:pg12-latest
```

> The above commands creates a TimescaleDB instanced named `timescaledb` (via the `--name` flag), on the network named `promscale-timescale` (via the `--network` flag), whose container will run in the background with the container ID printed after created (via the `-d` flag), with port-forwarding it to port `5432` on your machine (via the `-p` flag).

Reviewer: ```suggestion
The above commands create a TimescaleDB instanced named `timescaledb` (via the `--name` flag), on the network named `promscale-timescale` (via the `--network` flag), whose container will run in the background with the container ID printed after created (via the `-d` flag), with port-forwarding it to port `5432` on your machine (via the `-p` flag).
```

Reviewer 2: fixed

> > :WARNING: Note we set the `POSTGRES_PASSWORD` environment variable (using the `-e` flag) in the command above. Please ensure to replace `<password>` with the password of your choice for the `postgres` superuser.

Reviewer: ```suggestion
> :WARNING: We set the `POSTGRES_PASSWORD` environment variable (using the `-e` flag) in the command above. Please ensure to replace `<password>` with the password of your choice for the `postgres` superuser.
```

Reviewer 2: Fixed

> 

> > Note that for production deployments, you will want to fix the docker tag to a particular version instead of `pg12-latest`

Reviewer: ```suggestion
> For production deployments, you will want to fix the docker tag to a particular version instead of `pg12-latest`
```

Reviewer 2: Fixed

### Step 2: Install Promscale [](install-promscale)

> Since we have TimescaleDB up and running, let’s spin up a [Promscale][promscale-github], using the [Promscale docker image][promscale-docker-image] available on Docker Hub:

Reviewer: > let’s spin up a [Promscale][promscale-github]

"spin up Promscale" or "spin up a Promscale instance"?

Reviewer 2: changed to "Promscale instance"

```bash
docker run --name promscale -d -p 9201:9201 \
--network promscale-timescaledb \
timescale/promscale:latest
-db-uri postgres://postgres:<password>@timescaledb:5432/postgres?sslmode=allow
```

> In the `-db-uri` flag above, the second mention of `postgres` refers the the user we're logging into the database as, `<password>` is the password for user `postgres` and `timescaledb` is the name of the TimescaleDB container, installed in step 1. We can use the name `timescaledb` to refer to the database, rather than using its host address, as both containers are on the same docker network `promscale-timescaledb`.

Reviewer: ```suggestion
In the `-db-uri` flag above, the `postgres` after the double backslash refers to the user we're logging into the database as, `<password>` is the password for user `postgres`, and `timescaledb` is the name of the TimescaleDB container, installed in step 1. We can use the name `timescaledb` to refer to the database, rather than using its host address, as both containers are on the same docker network `promscale-timescaledb`.
```

Reviewer 2: Good clarifying language, thanks for adding

> >**:WARNING: Note**: The setting `ssl-mode=allow` is for testing purposes only. For production deployments, we advise you to use `ssl-mode=require` for security purposes. 

Reviewer: Correct the formatting by removing the highlight around "WARNING"

![image](https://user-images.githubusercontent.com/12288273/107257949-374d6100-6a09-11eb-8422-3e86729c56b5.png)

Reviewer 2: Fixed

>

> > Furthermore, note that value`<password>` should be replaced with the password you set up for TimescaleDB in step 1 above.

Reviewer: This should not be part of a warning, but rather part of the normal text

Reviewer 2: Fixed
